cli/src/pcluster/models/common.py

```diff
@@ -139,11 +139,20 @@ def validate(self, raise_on_error=False):
         self.__validators = sorted(self.__validators, key=operator.attrgetter("priority"), reverse=True)
 
         # Execute validators and add results in validation_failures array
-        for attr_validator in self.__validators:
+        for validator in self.__validators:
             # Execute it by passing all the arguments
             self._validation_failures.extend(
-                attr_validator.validator_class(raise_on_error=raise_on_error).execute(**attr_validator.validator_args)
+                validator.validator_class(raise_on_error=raise_on_error).execute(**validator.validator_args)
             )
+            for attr, value in self.__dict__.items():
```

Comment:
Why it is under the loop of `for validator in self.__validators`?

Reply:
Thanks!